apps/evi/db/schema.ts

```
@@ -1,5 +1,102 @@
+  sessionId: string
+  /** The thread, PR, or file the fact came from, when there is one to point at. */
+  url: string | null
+}
+
+export const people = pgTable('people', {
+  id: uuid().primaryKey().defaultRandom(),
+  tenantId: text().notNull(),
+  displayName: text().notNull().default(''),
+  role: text().$type<PersonRole>().notNull().default('visitor'),
+  createdAt: timestamp({ withTimezone: true }).notNull().defaultNow(),
+}, table => [
```

ESLint

[error] There should be no linebreak after '['.

(array-bracket-newline)